packages/components/src/Banner/Banner.tsx

```diff
@@ -117,6 +123,22 @@ export function Banner({
   }
 }
 
+function getBannerIcon(type: BannerType): IconNames | undefined {
```

Comment:
Is this only meant to override the icon if the user hasn't already set an icon? When I cycle between banner types the Icon doesn't update

Reply:
(Which I guess is a safe way to approach it, if someone has specified a preferred icon for a given use case, we leave that to them - otherwise use a sensible default)

Reply:
yes! you got it haha